senatus/src/api/submission/controllers/submission.ts

```diff
@@ -2,6 +2,96 @@
  * submission controller
  */
 
+import { connect } from '@colosseum/queue/amqp';
 import { factories } from '@strapi/strapi'
 
-export default factories.createCoreController('api::submission.submission');
+export default factories.createCoreController('api::submission.submission', ({ strapi }) => ({
+  async submit(ctx) {
+    try {
+      ctx.body = ctx.request.query;
+
+      const id = parseInt(ctx.request.query.id as string);
```

Comment:
Isn't it more convenient to make this method to also store a submission if it's not provided?

Reply:
It is, but I would leave it out for now.